zipline-bytecode/build.gradle.kts

```diff
@@ -9,10 +9,10 @@ plugins {
 }
 
 kotlin {
-  sourceSets.all {
```

Comment:
Not sure if this worked previously, but it currently doesn't opt in for the JVM plugin.

Reply:
Yeah I've only used this construct with the multiplatform plugin. Probably me copy/pasting too aggressively.